src/Data/NonEmpty.purs

```diff
@@ -117,3 +109,7 @@ instance traversableWithIndexNonEmpty
   => TraversableWithIndex (Maybe i) (NonEmpty f) where
   traverseWithIndex f (a :| fa) =
     NonEmpty <$> f Nothing a <*> traverseWithIndex (f <<< Just) fa
+
+instance foldable1NonEmpty :: Foldable f => SF.Foldable1 (NonEmpty f) where
```

Comment:
`SF`?

Reply:
San Francisco?

I was thinking _S_emigroup._F_oldable. But yeah, it's crappy. I changed it to `F1`.